src/PortfolioPlanning/Components/Plan/DependencyPanel.tsx

```diff
@@ -211,8 +213,8 @@ export class DependencyPanel extends React.Component<IDependencyPanelProps, IDep
                 <div className="item-list-row">
                     {item.data.showInfoIcon ? (
                         <Tooltip text={item.data.infoMessage}>
-                            <div className="info-icon">
-                                <Icon ariaLabel="Info icon" iconName="Info" />
+                            <div>
+                                <Icon ariaLabel="Info icon" iconName="Info" className="info-icon"/>

```

Comment:
Is this icon red?

Reply:
Nope, this icon is from azure-devops-ui. It's black by default, should I overwrite the style to red?

Reply:
Yes. Same as the way they change the icon color in their text field component: https://azdevinternal.azureedge.net/components/text-field#text-field-with-error

Reply:
Have changed the color to red.